src/Pages/Tables/Tables.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import SimpleTable from '../../Components/Table/SimpleTable';
+
+
+const styles = (theme) => ({
```

Comment:
Should be placed under Tables folder named as _styles.js?

Reply:
Yes! I'll update it. For now I only have updated the Components' styles.